content/security/docs/iam.md

```diff
@@ -90,7 +90,9 @@ By default when you provision an EKS cluster, the API cluster endpoint is set to
 + Configure public access with a set of whitelisted CIDR blocks and set private endpoint access to enabled. This will allow public access from a specific range of public IPs while forcing all network traffic between the kubelets (workers) and the Kubernetes API through the cross-account ENIs that get provisioned into the cluster VPC when the control plane is provisioned.
 
 ### Create the cluster with a dedicated IAM role
-When you create an Amazon EKS cluster, the IAM entity user or role, such as a federated user that creates the cluster, is automatically granted `system:masters` permissions in the cluster's RBAC configuration. This access cannot be removed and is not managed through the `aws-auth` ConfigMap. Therefore it is a good idea to create the cluster with a dedicated IAM role and regularly audit who can assume this role. This role should not be used to perform routine actions on the cluster, and instead additional users should be granted access to the cluster through the `aws-auth` ConfigMap for this purpose. After the `aws-auth` ConfigMap is configured, the role can be deleted and only recreated in an emergency / break glass scenario where the `aws-auth` ConfigMap is corrupted and the cluster is otherwise inaccessible. This can be particularly useful in production clusters which do not usually have direct user access configured.
+When you create an Amazon EKS cluster, the IAM entity user or role, such as a federated user that creates the cluster, is automatically granted `system:masters` permissions in the cluster's RBAC configuration. **This access cannot be removed and is not managed through the `aws-auth` ConfigMap.** Therefore it is a good idea to create the cluster with a dedicated IAM role and regularly audit who can assume this role. This role should not be used to perform routine actions on the cluster, and instead additional users should be granted access to the cluster through the `aws-auth` ConfigMap for this purpose. After the `aws-auth` ConfigMap is configured, the role can be deleted and only recreated in an emergency / break glass scenario where the `aws-auth` ConfigMap is corrupted and the cluster is otherwise inaccessible. This can be particularly useful in production clusters which do not usually have direct user access configured.
+
+Eksctl has guidance on the [minimum IAM policies](https://eksctl.io/usage/minimum-iam-policies/) required for creating an IAM role.
```

Comment:
Can you make this generic instead of linking to the eksctl documentation? eksctl is not as popular as other options like Terraform and CloudFormation. Also, eksctl requires greater permissions because in addition to creating the control plane, it can also create a VPC, OIDC endpoints, etc. Technically, the only permission you need to create a cluster is CreateCluster.

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html

Reply:
@Zvikan has implemented [iamlive](https://github.com/iann0036/iamlive) on terraform-aws-eks-blueprints to generate the minimum IAM policy document for creating clusters based on their configuration. https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/755

Perhaps this could be useful in-lieu of the eksctl doc

Reply:
Thanks @bryantbiggs. I saw that Blueprints is using iamlive. I like that approach as permissions will change depending on provisioning method used and the division of labor, i.e. who is responsible for what. Can you revise the PR with a few sentences on iamlive and how you would use it to create a minimal IAM policy.